tests/polling/infrequent/workflow_test.py

```diff
@@ -0,0 +1,31 @@
+import uuid
+
+import pytest
+from temporalio.client import Client
+from temporalio.testing import WorkflowEnvironment
+from temporalio.worker import Worker
+
+from polling.infrequent.activities import compose_greeting
+from polling.infrequent.workflows import GreetingWorkflow
+
+
+async def test_infrequent_polling_workflow(client: Client, env: WorkflowEnvironment):
+    if not env.supports_time_skipping:
```

Comment:
Do I need to do anything to make sure this gets run in time-skipping mode by CI?

Reply:
No, it's good as-is -- the test suite is run twice by the GitHub workflow: 
https://github.com/temporalio/samples-python/blob/58305c0fdf3f78e26ad21f14ada9da38547188b0/.github/workflows/ci.yml#L45-L47